valkyrie/lib/valkyrie/indexers/access_controls_indexer.rb

```diff
@@ -7,6 +7,7 @@ def initialize(resource:)
     end
 
     def to_solr
+      return {} unless resource.respond_to?(:read_users)
```

Comment:
I'm curious why only checking read_users is sufficient and not any of the other field accessors?